examples/ExampleApp.py

```diff
@@ -360,7 +362,7 @@ def __init__(self):
         self.ui.codeView.setLayout(self.codeLayout)
         self.hl = PythonHighlighter(self.ui.codeView.document())
         app = QtGui.QApplication.instance()
-        if QT_LIB in ['PyQt5', 'PySide2']:
+        if QT_LIB in ['PyQt5', 'PySide6']:
```

Comment:
this should be `QT_LIB in [PyQt5, PyQt6, PySide2, PySide6]`; but as we have deprecated Qt4 support, we can actually remove this if-statement/check, and un-indent the `paletteChanged.connect` call.

Reply:
Right, this was a leftover from replacing PySide2 with PySide6 for testing.